module/programs/helper.js

```diff
@@ -145,12 +144,21 @@ module.exports = class ProgramsHelper {
 					})
 					scopeData = _.omit(scopeData, keysCannotBeAdded)
 				}
-
+				const updateObject = {
```

Comment:
@Prajwal17Tunerlabs can you please add some comments.

Reply:
added comments to the code.